WooCommerce/UITestsFoundation/Screens/Products/ProductsScreen.swift

```diff
@@ -82,7 +82,8 @@ public final class ProductsScreen: ScreenObject {
     }
 
     public func selectProductType(productType: String) throws -> SingleProductScreen {
-        app.staticTexts[productType].tap()
+        let productTypeLabel = NSPredicate(format: "label CONTAINS[c] %@", productType)
+        app.staticTexts.containing(productTypeLabel).firstMatch.tap()
```

Comment:
Just thinking aloud: the buttons for product type selection have no identifiers (as of now), so identifiers can't be used (which would be justified, since this non-asserting function can potentially be used in tests for localizations). So, we can go for exact text values (as you did before). Is there a reason for less strict locator (only part of a string / case-insensitive) that you're using now?